infra/lambda_functions/rest/getdiagrams/app.js

```diff
@@ -11,16 +11,39 @@ const ddb = new AWS.DynamoDB.DocumentClient({
 });
 
 exports.handler = async (event) => {
-  let diagrams;
+  let diagrams = [];
 
   try {
     const diagramsResult = await ddb
       .scan({
         TableName: DIAGRAMS_TABLE_NAME,
-        ProjectionExpression: "diagramId",
+        ProjectionExpression: "diagramId, lastModified, revisionName",
```

Comment:
For now, scanning for every single diagram is OK, but we will need to limit this. perhaps to the first 50? We can adding paging if required. When we have users, we will obviously just get their diagrams. I think we can specify `Limit: 50`: https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Scan.html